index.js

```diff
@@ -346,6 +346,12 @@ BFFS.prototype.publish = function publish(spec, options, fn) {
             fn(err);
           });
         },
+        file.compressed && function (fn) {
+          cdn.upload(file.compressed, bff.key(file, 'file') + '.gz', (err, url) => {
+            file.compressedUrl = url;
```

Comment:
Do we want to set this regardless of if there is an error?

Reply:
@fritzmonkey my view was that it didnt matter much since error short circuits